osf/utils/tokens/handlers.py

```diff
@@ -12,34 +12,66 @@
 
 def registration_approval_handler(action, registration, registered_from):
     # TODO: Unnecessary and duplicated dictionary.
```

Comment:
Can we do anything about this TODO?

Reply:
I can't find the values (status messages) in that dictionary aren't anywhere else in osf.io (I tried to grep) and based on the git blame, the TODO is 5 years old so it seems outdated and I can remove it.